kytos/core/controller.py

```diff
@@ -582,6 +582,7 @@ def load_napp(self, username, napp_name):
             # This start method is inherited from the Threading class.
             # It is not directly defined/declared on the KytosNApp class.
             napp.start()
+            self.api_server.start_rest_api(napp)
```

Comment:
`s/start_rest_api/add_endpoints_from_decorators/g`

Reply:
I would rename it to `load_napp_endpoints` or `register_napp_endpoints`.